src/css/utilities.css

```diff
@@ -36,3 +36,17 @@
 
   cursor: pointer;
 }
+
+/**
+ * Don't use this class.
+ *
+ * Instead, find a better way to wrap content in whatever this class is
+ * applied to :/
+ */
+.HACK-overflow-x-scroll {
```

Comment:
So the `div` that applies this class should ideally not exist, but it alleviates the problem reported in #470 for now.